### DISCUSSION TITLE
Add support for font size and font family.

### DISCUSSION THREAD

**Discussion by engr-kishon** [awcodes/filament-tiptap-editor#300] [Ideas]
I want to suggest you to, add support for customizing font family and font size.

Follow below link for reference.
https://github.com/ueberdosis/tiptap/issues/388#issuecomment-833564783

**Comment by awcodes** [comment]
At this moment, I have no plans to include these in the editor as I feel they promote bad html practices and encourage content editors to "break" the branding of their sites.

If those are important to your app, then I would suggest making custom extensions to support them or use the tinymce plugin instead.

**Reply by engr-kishon** [reply]
Thanks, I used this plugin. https://github.com/mohamedsabil83/filament-forms-tinyeditor

**Comment by MNhat0308** [comment]
Hi Sir,
Is this feature currently being planned?

**Reply by awcodes** [reply]
It is not. See other comment for my reasoning. Sorry.

**Reply by MNhat0308** [reply]
Thanks sir!
